doc/particles.md

```diff
@@ -305,7 +316,7 @@ class RectComponent extends Component {
 
 To use Flare within Flame, use the [`flame_flare`](https://github.com/flame-engine/flame_flare) package.
 
-It will provide a class `FlareParticle` that is a container for `FlareActorAnimation`, it propagates `update` and `render` hooks to its child.
+It will provide a class called  `FlareParticle` that is a container for `FlareActorAnimation`, it propagates `update` and `render` methods to its child.
```

Comment:
```suggestion
It will provide a class called `FlareParticle` that is a container for `FlareActorAnimation`, it propagates the `update` and `render` methods to its child.
```